AfroApp/app/src/main/java/com/alice/afroapp/MainActivity.java

```diff
@@ -175,4 +175,12 @@ public void MentorSignUp(MenuItem item) {
         Intent intent= new Intent(MainActivity.this,AddProf.class);
         startActivity(intent);
     }
+

```

Comment:
This will cause conflicts because i wrote code in this class, it needs to be moved.